src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -978,7 +978,9 @@ def send_raw_request(cli_ctx, method, url, headers=None, uri_parameters=None,  #
         reason = r.reason
         if r.text:
             reason += '({})'.format(r.text)
-        raise CLIError(reason)
+        ex = CLIError(reason)
```

Comment:
What's the actual error type? Do we need use `AzCLIError` (maybe more specific error derived from `AzCLIError`)?

Reply:
I just kept the original design. We can refine the error type in the future for `send_raw_request`.